src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,55 @@
-import React from 'react';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+import { Post } from '../../types/Post';
+
+type Props = {
+  post: Post,
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  const commentsSection = () => {
```

Comment:
It would be great to name functions according to these rules
![image](https://github.com/mate-academy/react_static-list-of-posts/assets/96409563/2c0c217b-34f4-4930-87c2-6e60252d9c86)